pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -232,3 +249,62 @@ func (r *MigStorage) GetVSL(client k8sclient.Client) (*velero.VolumeSnapshotLoca
 
 	return nil, nil
 }
+
+// Determine if two secrets cloud secrets are equal.
+// Returns `true` when equal.
+func (r *MigStorage) EqualsCloudSecret(a, b *kapi.Secret) bool {
+	return reflect.DeepEqual(a.Data, b.Data)
+}
+
+// Get the cloud credentials secret by labels.
+func (r *MigStorage) GetCloudSecret(client k8sclient.Client) (*kapi.Secret, error) {
+	return GetSecret(
+		client,
+		&kapi.ObjectReference{
+			Namespace: "velero",
```

Comment:
Oh, one other thing I forgot. We should probably use the `veleroNs` const rather than hard-coding "velero" here.